lib/common/client/base.rb

```diff
@@ -34,6 +37,13 @@ def connection
             raise SecurityError, 'http client needs cookies stripped'
           end
 
+          if handlers.include?(Breakers::UptimeMiddleware)
+            return unless handlers.first == Breakers::UptimeMiddleware
+            raise BreakersImplementationError, 'Breakers should be the first middleware implemented.'
+          else
+            warn('Breakers is not implemented for this service.')
+          end
```

Comment:
Thanks for the suggestion @patrickvinograd

Reply:
This is great!